charts/orchestrator/templates/openshift-rhsso.yaml

```diff
@@ -0,0 +1,158 @@
+{{- if .Values.rhsso.enabled }}
+{{- $appLabel := printf "orchestrator-sso-%s" .Values.rhsso.realm.name }}
+{{- $namespace := .Values.rhsso.realm.namespace }}
+---
+apiVersion: v1
+kind: Namespace
+metadata:
+  name: {{ .Values.rhsso.namespace }}
+  meta.helm.sh/release-namespace: default
+spec:
+---
+apiVersion: operators.coreos.com/v1
+kind: OperatorGroup
+metadata:
+  name: sso-operator-group
+  namespace: {{ .Values.rhsso.namespace }}
+spec:
+  targetNamespaces:
+  - {{ .Values.rhsso.namespace }}
+  upgradeStrategy: Default
+---
+apiVersion: operators.coreos.com/v1alpha1
+kind: Subscription
+metadata:
+  name: rhsso-operator
+  namespace: {{ .Values.rhsso.namespace }}
+spec:
+  channel: stable
+  installPlanApproval: Automatic
+  name: rhsso-operator
+  source: redhat-operators
+  sourceNamespace: openshift-marketplace
+  startingCSV: {{ .Values.rhsso.startingCSV }}
+---
+apiVersion: keycloak.org/v1alpha1
+kind: Keycloak
```

Comment:
CRs can't be managed by the chart like this due to chicken and egg dependency with the subscription. Check how the CR for the RHDH is created.